aws/cloudformation/components/alarms.yml.erb

```
@@ -18,9 +18,11 @@
   DaemonMemoryUtilizationAlarm:
     Type: "AWS::CloudWatch::Alarm"
     Properties:
+<%    unless rack_env?(:adhoc) -%>
       AlarmActions:
         - !Sub "arn:aws:sns:${AWS::Region}:${AWS::AccountId}:CDO-<%= rack_env?(:production) ? 'Urgent' : 'LowPriority'%>"
-      AlarmDescription: Send page if daemon memory utilization exceeds 87% for 5 minutes
+<%    end -%>
+      AlarmDescription: Daemon memory utilization exceeds 87% for 5 minutes
```

super-tiny nit; I like cutting "send page", but I'd also like to update the tense of the resulting sentence:

```suggestion
      AlarmDescription: Daemon memory utilization exceeded 87% for 5 minutes
```